proto/imuachain/assets/v1/params.proto

```
@@ -1,12 +1,12 @@
 syntax = "proto3";
-package exocore.assets.v1;
+package imuachain.assets.v1;
 
 import "gogoproto/gogo.proto";
```

Buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)